src/ResourceManager/Resources/Commands.Resources/Providers/GetAzureProviderOperationCmdlet.cs

```diff
@@ -46,6 +47,11 @@ public class GetAzureProviderOperationCommand : ResourcesBaseCmdlet
         /// </summary>
         public override void ExecuteCmdlet()
         {
+            if (OperationSearchString == null)
```

Comment:
Note that you can simply set the default value in the property definition  to '*'' using

```c#
public string OperationSearchString { get; set; } = "*";
```

Reply:
If we have any test surface for thsi using '*' explicitly, let's change to using the default.

Reply:
Done and done.